samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction finding connectionless locator
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The direction finding connectionless locator sample application demonstrates Bluetooth LE direction finding reception.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix when operating in angle of arrival mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Overview
+********
+
+It uses Constant Tone Extension (CTE), that is received and sampled with periodic advertising PDUs.
+
+The sample supports two direction finding modes:
+
+* angle of arrival
+* angle of departure
```

Comment:
```suggestion
* angle of arrival (AoA)
* angle of departure (AoD)
```
Defining AoA and AoD here already, we can freely use them further down the file, without thinking about the definition :)

Reply:
Added AoA and AoD